src/wwwroot/js/genpage/gentab/imagehistory.js

```diff
@@ -127,6 +132,244 @@ function buttonsForImage(fullsrc, src, metadata) {
     return buttons;
 }
 
+function parseSearchQuery(query) {
+    const namespaces = ['lora', 'prompt', 'model', 'controlnet', 'seed', 'steps', 'cfgscale', 'aspectratio', 'width', 'height', 'sampler', 'videoframes', 'fps', 'date'];
```

Comment:
This is weirdly specific, why not just a generic metadata parameter by name check?